test-it/ViaTests/ByteArrayString.via

```diff
@@ -0,0 +1,87 @@
+
+define(HelloWorld dv(.VirtualInstrument (
+    Locals:
+    c(
+        ce(dv(a(.UInt8 *) ()) ba0)
+        ce(dv(a(.UInt8 *) (65 66 67 97 98 99)) ba1)
+        e(dv(.UInt16 0) encoding)
+        e(dv(.String) s1)
+        ce(dv(.String "ABCabc") s2)
+        e(dv(a(.UInt8 *)) ba2)
+        e(.ErrorCluster err)
+        e(dv(c(
```

Comment:
I think it might be good to add a few more tests cases with more representative UTF-8 strings (accented characters, non-English languages), such as those in the sample file in the utf8 source directory.

Reply:
The existing test UTF-8HelloWorld.via has UTF-8 strings and that is modified to test these two prims. We will do more testing in G CI side too.